pkg/tests/ossm/smoke_test.go

```diff
@@ -117,6 +107,26 @@ func TestSmoke(t *testing.T) {
 			})
 		})
 
+		t.NewSubTest(fmt.Sprintf("install smcp %s", toVersion)).Run(func(t TestHelper) {
+			t.Logf("This test checks whether SMCP %s deletion deletes all the resources", env.GetSMCPVersion())
+
+			t.LogStepf("Install SMCP %s", toVersion)
+			assertSMCPDeploysAndIsReady(t, toVersion)
+			assertRoutesExist(t)
+
+			t.LogStep("Install bookinfo pods and sleep pod")
+			app.InstallAndWaitReady(t, app.Bookinfo(ns), app.SleepNoSidecar(ns))
```

Comment:
Given that you install bookinfo in the subtest, you should also clean it up in the subtest instead of the parent test.

Reply:
Done